script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -410,7 +410,7 @@ As I watch the two of them out of the corner of my eye, I gaze idly at the scene
 -- Page 44, Offset 4962. Glued. Mods: @x.
 -- ……が大きいぶん、燃費が悪いのは自然の摂理っスよ」
 // I have no idea how to work in 摂理
- It's natural that the bigger you are, the less fuel-efficient you are."
+ It's natural that the bigger you are, the less fuel efficient you are."
```

Comment:
dash should be kept from what I've looked up.

Reply:
wait i'm confused now. i thought it was agreed on the other file that there isn't supposed to be a hyphen there

Reply:
I see. Ignore me then.

Reply:
It's true, we were unsure but apparently cambridge and collins dictionaries have `energy-efficient` hyphenated, so the same applies to this instance.

Reply:
i wasn't around when it was agreed though i'm also of the opinion of keeping the dash so i'm not really sure.

Reply:
this was something we talked about in person during a group pr. We decided based on some online article that non-hyphenated was the American version.